nautobot/core/api/metadata.py

```diff
@@ -71,6 +74,14 @@ def _get_field_properties(self, field: schema.SerializerType, name: str) -> Dict
                 result["required"] = type_map_obj.get("required", [])
             result["items"] = self._get_field_properties(field.child_relation, "")
             result["uniqueItems"] = True
+        elif isinstance(field, NautobotHyperlinkedRelatedField):
+            result["format"] = type_map_obj["format"]
+            model_options = field.queryset.model._meta
+            # Custom Keyword: modelName and appName
+            # This Keyword represents the model name of the uuid model
+            # and appName represents the app_name of the model
+            result["modelName"] = model_options.model_name
+            result["appName"] = model_options.app_label
```

Comment:
Do `modelName` and `appName`/`appLabel` need to also be added to the `ManyRelatedField` case?

Reply:
`ManyRelatedField` would recursively call the `_get_field_properties`, which would end up here `elif isinstance(field, drf_serializers.RelatedField):`